Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -1,14 +1,15 @@
 using System.Collections;
+using System.Linq;
 using Characters;
+using UI;
 using Unity.Services.Lobbies;
```

Comment:
Since we are already using Lobbies, we can start at Lobbies instead of saying 'Unity.Services.Lobbies'. We've done this a few times throughout.